packages/app/src/app/components/CodeEditor/Configuration/elements.ts

```diff
@@ -4,8 +4,8 @@ export const Container = styled.div`
   padding: 1rem;
   color: rgba(255, 255, 255, 0.8);
   box-sizing: border-box;
-
-  overflow: auto;
+  max-height: 100%;
+  background: red;
```

Comment:
😅?

Reply:
LOL! Removed, it was the one that didn't work ahah